packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -288,15 +288,28 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
   );
 
   /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
-  const onImageDownloadButtonClicked = useCallback((imageUrl: string, saveAsName: string): void => {
-    // Create a new anchor element
-    const a = document.createElement('a');
-    // Set the href and download attributes for the anchor element
-    a.href = imageUrl;
-    a.download = saveAsName || 'download';
-    // Programmatically click the anchor element to trigger the download
-    a.click();
-    a.remove();
+  const onImageDownloadButtonClicked = useCallback(async (imageUrl: string, saveAsName: string): Promise<void> => {
+    if (isIOS()) {
+      const reader = new FileReader();
+      reader.onload = () => {
+        window.open(imageUrl, '_blank');
+      };
+      const blob = await fetch(imageUrl).then((r) => r.blob());
```

Comment:
For composite, the imageUrl we pass in is already the blob url. Would fetch from a blob url work here?

Reply:
Yes, it does work, but this is a good point the reader really does nothing when I can just open the image url for the blob.